app/src/test/java/org/gnucash/android/test/unit/export/OfxExporterTest.java

```diff
@@ -55,27 +55,6 @@
 @Config(sdk = 21, shadows = {ShadowCrashlytics.class, ShadowUserVoice.class})
 public class OfxExporterTest {
 
-    private String mBookUID;
-    private SQLiteDatabase mDb;
-
-    @Before
-    public void setUp() throws Exception {
-        BooksDbAdapter booksDbAdapter = BooksDbAdapter.getInstance();
-        Book testBook = new Book("testRootAccountUID");
-        booksDbAdapter.addRecord(testBook);
-        mBookUID = testBook.getUID();
-        DatabaseHelper databaseHelper =
-            new DatabaseHelper(GnuCashApplication.getAppContext(), testBook.getUID());
-        mDb = databaseHelper.getWritableDatabase();
-    }
-
-    @After
-    public void tearDown() {
-        BooksDbAdapter booksDbAdapter = BooksDbAdapter.getInstance();
-        booksDbAdapter.deleteBook(mBookUID);
-        mDb.close();
-    }
-
     /**
```

Comment:
please revert to same as other export tester classes - export supposed to use temp book that added and then deletes it.

Reply:
I don't see the point of using temporary books and deleting them. Robolectric already provides a clean test environment for each test, so doing it again is just unnecessary. Additionally, it wouldn't work as other export tests, because `Account.toOfx` calls `getAccountsDbAdapter()` and uses the global accounts adapter for the "active" book, instead of the temporary book created in the test.